admin_manual/installation/server_tuning.rst

```diff
@@ -50,6 +50,14 @@ Caching improves performance by storing data, code, and other objects in memory.
 Memory cache configuration for the Nextcloud server must be installed and configured.
 See :doc:`../configuration_server/caching_configuration`.
 
+Compression
+-----------
+
+Enabling compression in your web server for plain filetypes like javascript, css and svg files
```

Comment:
> Enabling compression in your web server for JavaScript, CSS, and SVG files improves performance because fewer bytes need to be transferred to the clients.

Simpler version.

> all other plain filetypes are usually served by Nextcloud PHP

I'm not certain. We have the compression middleware for some requests: https://github.com/nextcloud/server/blob/master/lib/private/AppFramework/Middleware/CompressionMiddleware.php

If we don't know for sure, we better remove the second sentence.

Reply:
Done with https://github.com/nextcloud/documentation/pull/10644/commits/c9914dc9c1ae084fe54e1b0c9b71335c3697ce6b